### ISSUE TITLE
joomla-alert

### ISSUE THREAD

**Issue by astridx** [joomla/backend-template#332]
The header of joomla-alert has sometimes a space on the right side:
![Menus  New Item   test   Administration(2)](https://user-images.githubusercontent.com/9974686/60651122-d2518280-9e45-11e9-9309-be6bb4557800.png)
![New   test   Administration](https://user-images.githubusercontent.com/9974686/60649142-bba92c80-9e41-11e9-93a9-643871502ff5.png)
![Extensions  Discover   test   Administration](https://user-images.githubusercontent.com/9974686/60649143-bba92c80-9e41-11e9-88e8-f8d5316ce2a8.png)

I would like to have it like this alert:
![Edit   test   Administration](https://user-images.githubusercontent.com/9974686/60649138-bb109600-9e41-11e9-86fa-3ed9c0446a51.png)

**Comment by brianteeman** [comment]
I can not replicate this when using the latest css on the "edit-screen-approvement" branch using google chrome on windows

**Comment by astridx** [comment]
@brianteeman You are right. On chrome it is fine. But on firefox 67.0.4 (64 Bit) the problem persists. I am working on ubuntu 18.04.
But perhaps it is a firefox problem. I have the space in the newly created help button, too
![PROJECT_NAME   Categories   test   Administration](https://user-images.githubusercontent.com/9974686/60720328-b8d63680-9f2a-11e9-831b-3e6696f58b43.png)

**Comment by brianteeman** [comment]
ok well its good to know that there is an issue to resolve and to test it we have to test with firefox

**Comment by brianteeman** [comment]
@astridx I think I might no where the problem lies and why its ok on some and not on others. Can you test this.
1. Create an article - dont enter a title and try save&close
I believe this will give you an error message and there will be the white space on the right
2. Create an article with a title and try save&close
I believe this will take you back to the article manager with a success message and there will not be the white space.

If you can hopefuly confirm this then I know where the problem is

**Comment by astridx** [comment]
I can confirm this. And it was in the past the same. The green message is Ok. But the red error has the white space on the right side.

**Comment by brianteeman** [comment]
ok so the problem is that the messages are generated by different code and there must be a slight difference. the success message is generated in messages.php and the other one in core.js

**Comment by bembelimen** [comment]
Mhh, it does not happen in my regular resolution, but if I play arround with resizing (CTRL and [+]) it sometimes appears.
